### DISCUSSION TITLE
"Oldest First" seeking seems to have some issue

### DISCUSSION THREAD

**Discussion by adformkyadav** [provectus/kafka-ui#3636] [Q&A] [answered]
Hi,

Here is the topic "geo.cities" which has around 13849547 messages in it. Here is the screenshot for topic overview
<img width="1435" alt="topic_details" src="https://user-images.githubusercontent.com/119411991/230769101-c4186d05-a141-4c8e-ba89-b5c3e034d5c8.png">

When I try to see messages it says "No messages found"
<img width="1436" alt="oldest_first" src="https://user-images.githubusercontent.com/119411991/230769209-1c6be0ef-9669-41e4-a53e-793318eef7de.png">

But I can see all the messages by selecting "Newest First"
<img width="1440" alt="newest_first" src="https://user-images.githubusercontent.com/119411991/230769479-d854993b-6064-4e9f-94b3-1a42073662e2.png">

So, there are messages present but question is why it isn't showing.

One thing I noticed is that, on UI it says "Assigning partitions" and then "Polling" and after 3001 ms it says no message found
So I guess it is getting timedout and  trying to fetch messages only for 3 seconds (maybe starting from offset 0) and if doesn't find anything it says there is no messages

I would like to confirm that this topic has "cleanup.policy" COMPACT and oldest message it has at offset 7817926(and second oldest is at offset 13260613)

So this might be the reason for the timeout.

But again we do know that there are messages in this topic beforehand as we are showing it on "Overview" section of topic, so I'm guessing this issue can be fixed.

**Comment by Haarolean** [comment, marked as answer]
Hi, can you try tuning these properties?

https://github.com/provectus/kafka-ui/issues/3485#issuecomment-1489898841

Let me know how it goes
